content/docs/command-reference/exp/show.md

```diff
@@ -235,8 +260,39 @@ $ dvc exp show --all-commits --include-params=featurize --sort-by=auc --sort-ord
 └───────────────────────┴──────────────┴─────────┴────────────────────────┴──────────────────┘
 ```
 
-Note that in the final example, Git commits remain in chronological order. The
+Note that in this example, Git commits remain in chronological order. The
 sorting only applies to experiment groups (sharing a parent commit).
 
+## Example: Parallel coordinates plot (PCP)
+
+To generate an interactive [PCP](#parallel-coordinates-plot) based on the experiments
+and their parameters:
+
+```dvc
+$ dvc exp show --all-branches --pcp
+```
+
+![](/img/ref_pcp_default.png) _Parallel Coordinates Plot_
+
+The color of the lines is determined by the value of the column you select with
+`--sort-by`:
+
+```dvc
+$ dvc exp show --all-branches --pcp --sort-by roc_auc
+```
+
+> When `--sort-by` is not supplied, the color of the lines will be determined by
+> the `Experiment` column.
+
+![](/img/ref_pcp_sortby.png) _Colorized by roc_auc_
+
+Combine with other flags for further filtering:
+
+```dvc
+$ dvc exp show --all-branches --pcp --sort-by roc_auc --exclude-metrics avg_prec
```

Comment:
To avoid horizontal scroll when rendered (committing):
```suggestion
$ dvc exp show --all-branches --pcp --sort-by roc_auc
               --exclude-metrics avg_prec
```